src/rebar_utils.erl

```diff
@@ -70,7 +70,9 @@
          info_useless/2,
          list_dir/1,
          user_agent/0,
-         reread_config/1]).
+         reread_config/1,
+         get_proxy_auth/0,
+         set_proxy_auth/1]).
```

Comment:
Sure, there is no problem to remove `set_proxy_auth` from the export list. I'll fix it.